### PR TITLE
Make task

### PR DESCRIPTION
**Pull request by alex39292**
JS functions task

### CODE REVIEW

File: secondTask.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function determineMyName(name) {
+    return ( name == 'aliaksandr') ? "Hello " + name : "It's not my name";
```
Comment (SiarheiRylach) [inline secondTask.js]: Better to use strict comparison

File: fourthTask.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function selectFromArray(...arrgs) {
+    let array = [];
+    for (i = 1; i < arrgs.length; i+=2) {
+        array.push(arrgs[i]);
+    }
+    return String(array)
```
Comment (SiarheiRylach) [inline fourthTask.js]: you should return a new array, not a string

File: secondTask.js (hunk 1 above)
Comment (alex39292) [reply secondTask.js]: fixed

File: fourthTask.js (hunk 2 above)
Comment (alex39292) [reply fourthTask.js]: Fixed

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**